bin/flutter

```diff
@@ -161,7 +161,7 @@ set +e
 # The VM exits with code 253 if the snapshot version is out-of-date.
 # If it is, we need to snapshot it again.
 EXIT_CODE=$?
-if [ $EXIT_CODE != 253 ]; then
+if [[ $EXIT_CODE != 253 ]]; then
```

Comment:
`-ne` for integer comparisons `!=` for strings.

Reply:
updated accordingly.